packages/schema/src/dialects/postgres.ts

```diff
@@ -61,7 +61,7 @@ export default class Postgres extends KnexPostgres implements SchemaInspector {
           AND pg_attribute.attrelid = pg_class.oid
           AND pg_attribute.attnum = ANY (pg_index.indkey)
           AND indisprimary
-          AND indnkeyatts = 1
+          AND indnatts = 1
```

Comment:
Looks like it will not work if the index contains non-key columns.
Consider `array_length(pg_index.indclass, 1) = 1` or `array_length(pg_index.indcollation, 1)`

Reply:
I don't believe a primary key index can include non-key columns, so is that really an issue ?